packages/cognito/preTokenGeneration/groups.test.ts

```diff
@@ -0,0 +1,160 @@
+import { PreTokenGenerationTriggerEvent } from "aws-lambda";
```

Comment:
Really love having tests now that we have sufficiently complex logic here

Reply:
Yeah! I was really happy with `jest`'s `each` function too! I was able to use the SAME test for SAML and OIDC without any duplication! I like that it's capable of decomposing the given objects into parameters to the test function cleanly too.